src/App.scss

```diff
@@ -1,3 +1,7 @@
 iframe {
   display: none;
 }
+
+.button {
+  min-width: 43px;
```

Comment:
seems like a magic number to me.
Usually the props like width, height, paddings, margins etc. should be theoretically divided by 4 without remainder

Reply:
will do for the training project 👍 
Will be much easier with sass on the real one